sdk/storage/AzureStorageBlob/Source/Operations/BlockOperation.swift

```diff
@@ -84,8 +84,10 @@ internal class BlockOperation: TransferOperation {
                     self.notifyDelegate(withTransfer: parent)
                 case let .failure(error):
                     var err = error
-                    if let pipelineError = error as? PipelineError {
-                        err = pipelineError.innerError
+                    switch error {
```

Comment:
I'm unclear what this is doing. It seems like it's a no-op?

Reply:
It was some wonky logic. Based on other changes, I was able to simplify this so it just says `parent.error = error`.